docs/config.rst

```diff
@@ -46,6 +58,7 @@ Those configuration options are documented below:
 .. describe:: environment
 
     Track the environment name inside Sentry.
+    Optionally, use ``SENTRY_ENVIRONMENT`` environment variable.
```

Comment:
It has been suggested we should default to `NODE_ENV` – super convenient, probably worth exploring.